<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSoftwareBinningclass</class>
 <widget class="QWidget" name="ImageSoftwareBinningclass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>706</width>
    <height>533</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>706</width>
    <height>533</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ImageSoftwareBinningclass</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ImagePseudoColor.ico</normaloff>ImagePseudoColor.ico</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>531</width>
     <height>531</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_binningImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string>binningImage</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_HBinning">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Custom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_originalImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="text">
    <string>originalImage</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>8×8 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_VBinning">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>StartBinning</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_16">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>16×16</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4×4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveClose">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE&amp;&amp;CLOSE</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2×2 </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_resize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Resize to original size</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
